core/src/test/java/io/questdb/griffin/SqlCodeGeneratorTest.java

```diff
@@ -782,7 +782,7 @@ public void testEqGeoHashWhenOtherIsStr2() throws Exception {
     @Test
     public void testFailsForLatestByOnSubQueryWithNoTimestampSpecified() throws Exception {
         assertFailure(
-                "with tab as (x where b in ('BB')) tab latest by b",
+                "with tab as (x where b in ('BB')) select * from tab latest by b",
```

Comment:
```suggestion
                "with tab as (x where b in ('BB')) select * from tab latest on b",
```
I don't think this matters for this PR but probably better to remove latest by?

Reply:
this is a "negative" test, which verifies behaviour on invalid SQL